src/Tizen.NUI.Components/Utils/StyleManager.cs

```diff
@@ -89,26 +60,48 @@ public string Theme
         {
             get
             {
-                return currentThemeName;
+                return ThemeManager.CurrentTheme.Id;
             }
-
             set
             {
-                if (value != null && currentThemeName != value)
+                if (value == null) return;
+
+                var key = value.ToUpperInvariant();
+
+                if (key == Theme.ToUpperInvariant()) return;
```

Comment:
on mobile or tv profile, Theme is null.
it causes crash System.NullReferenceException.